release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1313,6 +1317,20 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-olm-bug-fixes_{context}"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Previously, clusters with many custom resources (CRs) experienced timeouts from the API server and stranded updates where the only workaround was to uninstall and then reinstall the stranded Operators. This occurred because OLM evaluated potential updates by using a dynamic client lister. With this fix, OLM uses a paging lister for custom resource definitions (CRDs) to avoid timeouts and stranded updates. (link:https://issues.redhat.com/browse/OCPBUGS-41549[*OCPBUGS-47549*])
+
+* Previously, catalog source pods could not recover from a cluster node failure when the `registryPoll` was unset. With this fix, OLM updates its logic for checking for dead pods, and as a result, catalog source pods now recover from node failures as expected. (link:https://issues.redhat.com/browse/OCPBUGS-39574[*OCPBUGS-39574*])
```

Comment:
anything in backticks should be followed by what it is, similar to how it's done in the [quick markup reference table](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). the separate sentences at the end is just a suggestion because it was a little long:
```suggestion
* Previously, catalog source pods could not recover from a cluster node failure when the `registryPoll` parameter was unset. With this fix, OLM updates its logic for checking for dead pods. As a result, catalog source pods now recover from node failures as expected. (link:https://issues.redhat.com/browse/OCPBUGS-39574[*OCPBUGS-39574*])
```